pkg/ui/src/views/reports/containers/customChart/customChart.styl

```diff
@@ -35,6 +35,15 @@
     &:hover
       background-color $dropdown-hover-color
 
+    .Select-menu-outer
+      margin-top $spacing-base
+
+  .Select-control
+    width 100%!important
```

Comment:
any way to avoid using `!important`?

Reply:
I'd like to, but that's the only way to override defined `!important` in `shame.styl` file.
I would file a separate issue to refactor `Dropdown` component and other places that use shame styles so we could decrease this chaos with style pollution.

Reply:
Filed another issue for this: #50431